src/components/Annotation/Annotation.scss

```diff
@@ -0,0 +1,126 @@
+@import '../../common.scss';
+
+.ba {
+    .ba-annotation-container {
+        @import '~box-react-ui/lib/styles/common/inline-notifications';
+
+        border-bottom: 1px solid $seesee;
+        position: relative;
+        z-index: 1;
+
+        &:last-child {
+            border-bottom: 0;
+        }
+
+        .ba-annotation-error {
+            padding: 14px 15px;
+
+            .ba-annotation-error-action {
+                color: $king-crimson;
+                display: block;
+                margin-top: 5px;
+                text-decoration: underline;
+            }
+        }
+    }
+
+    .ba-annotation {
+        display: flex;
+        margin-top: 23px;
```

Comment:
Magic numbers should generally be avoided. Why does this need to be offset 23 pixels?

Reply:
Not entirely sure why this offset was picked. This matches the offset for Comments in the Activity Feed https://github.com/box/box-ui-elements/blob/master/src/components/ContentSidebar/ActivityFeed/comment/Comment.scss#L22